src/scaladoc/scala/tools/nsc/doc/base/CommentFactoryBase.scala

```diff
@@ -183,11 +183,11 @@ trait CommentFactoryBase { this: MemberLookupBase =>
 
   /** The start of a Scaladoc code block */
   private val CodeBlockStartRegex =
-    new Regex("""(.*?)((?:\{\{\{)|(?:\u000E<pre(?: [^>]*)?>\u000E))(.*)""")
+    new Regex("""(.*?)((?:\{\{\{)|(?:<pre(?: [^>]*)?>))(.*)""")
 
   /** The end of a Scaladoc code block */
   private val CodeBlockEndRegex =
-    new Regex("""(.*?)((?:\}\}\})|(?:\u000E</pre>\u000E))(.*)""")
+    new Regex("""(.*?)((?:\}\}\})|(?:</pre>))(.*)""")
```

Comment:
What's this change about?  (Have I asked this already?)

Reply:
I replaced it with the literal `\u000E` character, which looks like `` (rendering varies). for some reason, github isn't rendering it in diff mode, but if you look at the full file without diff, it's there (https://github.com/scala/scala/blob/8447c4274ea9f8ac10dad6a04b93f47a11fb99cb/src/scaladoc/scala/tools/nsc/doc/base/CommentFactoryBase.scala#L186-L190)

Reply:
I see, thanks!

Reply:
Is this because we don't know how "\u" is processed any more? There's also "\x0e".
```
raw"\x09".r.findAllMatchIn("aaa\tbbb\tccc").map(_.start).toList
```

Reply:
I fear that GitHub isn't the only tooling that isn't going to handle an embedded invisible control character well. I think it would really be better to use an escape. No opinion on `\x` vs `\u`, but note that since https://github.com/scala/scala/pull/8282 , `\u` is no longer weird/special

Reply:
the reason I replaced it was because of a warning that said to use a literal; I have no particular opinion on it otherwise

Reply:
I tried it and it is awkward. It's stuck half-way, like Winnie-the-Pooh. I'd recommend `\x`. I'm going to try `\X`; there's that other PR about mkstring with unicode points.
```
scala> "a\u0009b"
val res0: String = a    b

scala> raw"a\u0009b"
            ^
       warning: Unicode escapes in raw interpolations are deprecated. Use literal characters instead.
val res1: String = a    b
```

Reply:
what on earth are these random control characters doing in the regexes in the first place? surely Scaladoc doesn't actually use control-N for anything? is it possible somebody meant control-M, which is carriage return? control-N is "SO (SHIFT OUT)", does anybody under the age of 80 even know what that is?

Reply:
let's see if all the tests still pass if we just take it out: #9359